New package: AliasTables v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: AliasTables
- Repository: https://github.com/LilithHafner/AliasTables.jl
- Created by: @LilithHafner
- Version: v0.1.0
- Commit: 04a3ac9cda0ccc7d3c5db545ec30c4071a71fe07
- Reviewed by: @LilithHafner
- Reference: https://github.com/LilithHafner/AliasTables.jl/commit/04a3ac9cda0ccc7d3c5db545ec30c4071a71fe07#commitcomment-140345656
- Description: An efficient sampler for discrete random variables
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6edf82f5b417b77205e96ca0859b6cdd45ac653f74f402c146079652170a31f485371214feb158abe29e1be7ede61d0057e7fd489c798f6628ca06378f76725152f6aa2c23b9e9c48e9247097c327c33ecc079963a48f1a5836bb2988b0d716793d17b3954c0ca5a48f6f0f6882c19b18ccf73d9a61d8755f3be836946131a79334f49a8853aaa88a70cc3bbb76222add8fe460c64ac52e14590f8713344bf274170c57837c87e69285313aa99ccb4f90 -->

LilithHafner commented:
I think name similarity is okay, but I'll wait for someone else to agree before tagging for manual merge to override the auto-merge failure due to needing Julia 1.11.

JuliaRegistrator committed:
New package: AliasTables v0.1.0

UUID: 66dad0bd-aa9a-41b7-9441-69ab47430ed8
Repo: https://github.com/LilithHafner/AliasTables.jl.git
Tree: d3d46667f9b38f84389cf07a8eeb45356260aee8

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
